EduNLP/I2V/i2v.py

```diff
@@ -12,6 +12,8 @@
 
 class I2V(object):
     """
+    It just a api, so you shouldn't use it directly. \
```

Comment:
Please check whether the comments should put inside of the __init__() or outside.